crates/wasm/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export CFLAGS_wasm32_unknown_unknown="-I$(pwd)/wasm-sysroot -Wbad-function-cast -Wcast-function-type -fno-builtin" RUSTFLAGS="-Zwasm-c-abi=spec"
+export AST_GREP_RULE='{"id":"fix-wasm-js-node","language":"javascript","rule":{"pattern":"module_or_path = fetch(module_or_path);","inside":{"pattern":"function __wbg_init($$$) {$$$}","stopBy":"end"}},"fix":"if (!!process.versions.node) {\n  const fs = await import(\"fs/promises\");\n  module_or_path = fs.readFile(module_or_path);\n} else {\n  module_or_path = fetch(module_or_path);\n}\n"}'
+
+# Build with exhaustive-lang feature
+export OUT_DIR=pkg-exhaustive
+wasm-pack build -s ast-grep --release --target web --out-dir $OUT_DIR --features "exhaustive-lang" \
+	-Z build-std=panic_abort,std -Z build-std-features=panic_immediate_abort
+
+# Make the WASM module work with Node.js as Node does not support fetch for file:// URLs
+cargo run --manifest-path ../cli/Cargo.toml -- scan --inline-rules "$AST_GREP_RULE" -U $OUT_DIR/ast_grep_wasm.js
+
+sed -i ".bak" -e 's/"name": "@ast-grep\/ast-grep-wasm"/"name": "@ast-grep\/wasm"/g' $OUT_DIR/package.json
+rm $OUT_DIR/package.json.bak
+
+# Build with small-lang feature
+export OUT_DIR=pkg-small
+wasm-pack build -s ast-grep --release --target web --out-dir $OUT_DIR --features "small-lang" \
+	-Z build-std=panic_abort,std -Z build-std-features=panic_immediate_abort
```

Comment:
i tried this build command but it does not work the `-Z` flag is interpreted as folder

Reply:
i don't know what happend but it turns out the command it self is okay if I added features

Reply:
yeah i've noticed that too. really strange.